.env.example

```diff
@@ -12,6 +12,7 @@ DEFAULT_RATE_LIMIT =
 TIER_1_RATE_LIMIT =
 TIER_2_RATE_LIMIT =
 TIER_3_RATE_LIMIT =
-LIMIT_DURATION = 
-HBAR_RATE_LIMIT_TINYBAR = 
-HBAR_RATE_LIMIT_DURATION = 
\ No newline at end of file
+LIMIT_DURATION =
+HBAR_RATE_LIMIT_TINYBAR =
+HBAR_RATE_LIMIT_DURATION =
+ETH_GET_LOGS_BLOCK_RANGE_LIMIT =
```

Comment:
Please add Helm chart configuration option.
See https://github.com/hashgraph/hedera-json-rpc-relay/pull/578 on how it was recently done